packages/amplify_core/example/readme.md

```diff
@@ -0,0 +1,3 @@
+# amplify_core
+
+This is an internal package containing shared types and methods for use in amplify packages. 
```

Comment:
Capitalize filename to `README.md` to match others.

I would change the text to be closer to what's in README as well:

```suggestion
A core package containing base types and utilities shared across Amplify packages. 
```